packages/core/src/tools/stream.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { isIE } from './browserDetection'
+import { readLimitedAmountOfBytes } from './stream'
+
+describe('stream', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ ‏use test subject name directly
```suggestion
describe('readLimitedAmountOfBytes', () => {
```